app/Http/Controllers/API/AccountApiController.php

```
@@ -263,6 +266,50 @@ public function getItem(AccountEntity $accountEntity): JsonResponse
+        if (! $accountEntity->isAccount()) {
+            return response()->json([
+                'message' => __('This account entity is not an account.'),
+            ], Response::HTTP_BAD_REQUEST);
+        }
+
+        $validated = $request->validate([
+            'preferred_file_import_profile_id' => [
+                'nullable',
+                'integer',
+                Rule::exists('file_import_profiles', 'id'),
+                function (string $attribute, mixed $value, Closure $fail) use ($request): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)